src/System.Memory/src/System/PinnableReferenceHelpers.Layouts.Microsoft.cs

```diff
@@ -0,0 +1,35 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
What does the ".Microsoft" in the filename refer to?  If it's about a particular platform, we should use a word or words that best describes the target.